src/features/operator/Market/MarketWithdraw/index.tsx

```diff
@@ -118,6 +129,170 @@ export const MarketWithdraw = (): JSX.Element => {
     }
   };
 
+  const baseToPreferredCurrency = React.useMemo(() => {
+    // Wait for prices to load (or error out) before displaying conversions
+    if (isLoadingPrices || isErrorPrices) {
+      return '';
+    }
+
+    let amount = Big(0);
+    try {
+      amount = Big(balanceBaseAmount);
+    } catch {
+      // ignore user typos, just leave the amount as 0
+    }
+    const assetId = selectedMarket.baseAsset?.asset_id || '';
+    const assetName = selectedMarket.baseAsset?.ticker || 'unknown';
+
+    let amountInFiatOrLBTC = '';
+    if (isLbtcAssetId(selectedMarket.baseAsset?.asset_id || '', network)) {
+      amountInFiatOrLBTC = formatLbtcUnitToSats(amount.toNumber(), lbtcUnit);
+      amountInFiatOrLBTC = formatSatsToUnit(Number(amountInFiatOrLBTC), 'L-BTC', assetId, network);
```

Comment:
`balanceBaseAmount` should already be given by user in LBTC favorite unit

Reply:
Yep. The issue is the CoinGecko prices are for "1 whole L-BTC unit".

So we need to go from:
UserFavoriteUnit -> SatsUnit
SatsUnit -> L-BTCUnit

AFAIK there's not a "FormatLbtcUnitToUnit" function.